elastic/integrations
# Removing detection rules from `dga` package

## What does this PR do?
This PR moves detection rules associated with DGA (dga) out of the integration package. 

## Checklist

## How to test this PR locally
- Spin up a local Elasticsearch cluster
- Verify that the package version shown is 2.0.0, Subscription is `platinum` and License is `Elastic-2.0`
- Install the Domain Generation Algorithm Detection assets
- Verify that no detection rules with the `DGA` tag show up on the Rules Management page under Security

## Related Issues
Not linking since these are internal issues

## Screenshots
- Version, Subscription and License is as expected
<img width="1274" alt="Screen Shot 2023-09-14 at 2 20 59 PM" src="https://github.com/elastic/integrations/assets/30438249/065d1f2f-2984-4507-8392-e09184e01ea7">
- No rules displayed upon installing the package assets
<img width="1761" alt="Screen Shot 2023-09-14 at 2 21 27 PM" src="https://github.com/elastic/integrations/assets/30438249/10aabf1c-dbf5-416d-ae4a-38c23ddfb4b0">
